internal/client/client.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/HewlettPackard/hpegl-pcbe-terraform-resources/internal/auth"
+	"github.com/HewlettPackard/hpegl-pcbe-terraform-resources/internal/defaults"
+	"github.com/HewlettPackard/hpegl-pcbe-terraform-resources/internal/sdk/async"
+	"github.com/HewlettPackard/hpegl-pcbe-terraform-resources/internal/sdk/systems"
+	"github.com/HewlettPackard/hpegl-pcbe-terraform-resources/internal/sdk/virt"
+	"github.com/microsoft/kiota-abstractions-go/authentication"
+	"github.com/microsoft/kiota-http-go"
```

Comment:
While looking at the very very long line i noticed, does this need to be `nethttplibrary "whatever"` to compile?

Reply:
The ci build job is passing.

Reply:
spooky, wonder how that works

Reply:
oh that's the name of it's 'main' package, TIL